tools/api-markdown-documenter/src/api-item-transforms/default-implementations/TransformApiInterface.ts

```diff
@@ -63,32 +62,47 @@ export function transformApiInterface(
 	const filteredChildren = filterChildMembers(apiInterface, config);
 	if (filteredChildren.length > 0) {
 		// Accumulate child items
-		const constructSignatures = filterByKind(apiInterface.members, [
-			ApiItemKind.ConstructSignature,
-		]).map((apiItem) => apiItem as ApiConstructSignature);
-
-		const allProperties = filterByKind(apiInterface.members, [
-			ApiItemKind.PropertySignature,
-		]).map((apiItem) => apiItem as ApiPropertySignature);
+		const constructSignatures: ApiConstructSignature[] = [];
+		const allProperties: ApiPropertyItem[] = [];
+		const callSignatures: ApiCallSignature[] = [];
+		const indexSignatures: ApiIndexSignature[] = [];
+		const methods: ApiMethodSignature[] = [];
+		for (const child of filteredChildren) {
+			switch (child.kind) {
+				case ApiItemKind.ConstructSignature: {
+					constructSignatures.push(child as ApiConstructSignature);
+					break;
+				}
+				case ApiItemKind.Property:
```

Comment:
This is the crux of the fix here. There are cases where children are `ApiProperty`s, not just `ApiPropertySignature`s.